game/menu.cc

```diff
@@ -15,6 +15,11 @@ std::string MenuOption::getName() const {
 	return "";
 }
 
+std::string MenuOption::getVirtName() const {
+	if (!virtualName.empty()) return virtualName;
```

Comment:
Can be made into a single return statement:
`return !virtualName.empty() ? virtualName : std::string();`